pkg/controller/podautoscaler/replica_calculator_test.go

```diff
@@ -446,20 +446,6 @@ func TestReplicaCalcDisjointResourcesMetrics(t *testing.T) {
 	tc.runTest(t)
 }
 
-func TestReplicaCalcMissingContainerMetricError(t *testing.T) {
```

Comment:
What do you think about converting this test to one that returns a proper replica number when the container is missing from some pods?

Reply:
Works for me; there's a similar test here and I just added in another one that includes metrics for one container in two pods but not the one we're asking for